src/caselawclient/Client.py

```diff
@@ -56,6 +56,11 @@
 )
 
 env = environ.Env()
+
+# Requests timeouts: https://requests.readthedocs.io/en/latest/user/advanced/
+CONNECT_TIMEOUT = float(os.environ.get("CONNECT_TIMEOUT", "3.05"))
+READ_TIMEOUT = float(os.environ.get("READ_TIMEOUT", "27"))
```

Comment:
This can be a lot more aggressive I think. If we haven't had a response back in... maybe five seconds we should consider showing the user an error.

If we catch timeout errors and re-raise as something more specific to us timing out when talking to MarkLogic then we can get a better picture in Rollbar as well.